.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+# CircleCI 2.0 configuration file for running Python on macOS.
+#
+# Based on these v1.0 steps:
+# https://pewpewthespells.com/blog/building_python_on_circleci.html
+#
+version: 2
+
+jobs:
+  build:
+    macos:
+      # We don't use the xcode, but we need to put something here.
+      xcode: "9.0"
+
+    working_directory: ~/repo
+
+    steps:
+      # Get the source.
+      - checkout
+
+      - run:
+          name: prepare environment
```

Comment:
Can this say "Prepare macOS environment" so there's some indication of the build's platform on its page?

Reply:
Sure :)...I am  testing a bit with Circle-CI to see if I can control the way commit status is reported